src/components/multi-autocompleter/multi-autocompleter.test.jsx

```diff
@@ -257,6 +257,34 @@ describe('<MultiAutocompleter>', () => {
       expect(await findSelectedOption('test label', 'Foo')).toBeInTheDocument();
       expect(await findSelectedOption('test label', 'Bar')).toBeInTheDocument();
     });
+
+    it('accepts an array of ids', async () => {
+      const { rerender } = render(<MultiAutocompleter {...requiredProps} value={[]} />);
+
+      expect(await querySelectedOption('test label', 'Foo')).not.toBeInTheDocument();
+
+      rerender(<MultiAutocompleter
+        {...requiredProps}
+        value={['55']}
+      />);
+
+      const foo = await findSelectedOption('test label', 'Foo');
+      expect(foo).toBeInTheDocument();
+    });
+
+    it('accepts a single id', async () => {
```

Comment:
I'm not sure if we want to keep this. A multi-completer should always accept a list of things since it is suppose to select multiple things. 

What is the bigmaven use-case for this kind of API?

Reply:
Yeah. Why not keep it?

Reply:
It's a bit easier to reason about when we know the `value` prop is always an array (and preferably an array of _something_). 

It also seems like `*_ids` fields should always be arrays. What is an example network payload where it returns a single id vs a list of ids? I bet there might be a bug in the API responses (or API parsing).

Reply:
`approver_id` is one example